packit.yaml

```diff
@@ -0,0 +1,34 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: fedora-messaging.spec
+
+actions:
+    post-upstream-clone: "wget https://src.fedoraproject.org/fork/t0xic0der/rpms/fedora-messaging/raw/f35-300-src/f/fedora-messaging.spec -O fedora-messaging.spec"
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-35
+      branch: packit
+      owner: t0xic0der
+      project: fedora-messaging
+  - job: copr_build
```

Comment:
lets add some space between each job, we're going to have a few so it will be easier to read.

Reply:
Yep, added some spaces and some comments too.